MiniAn/minian_parameters.py

```diff
@@ -143,6 +143,16 @@ def __init__(self, danse_parameters):
             "jac_thres": 0.2
         }
 
+        self.params_cross_reg = {}
+        if self.parameters[defs.Parameters.danse.CROSS_REG]:
+            self.params_cross_reg = {
+                "cross_reg"     : self.parameters[defs.Parameters.danse.CROSS_REG],
```

Comment:
I don't think this parameter is needed now. You can check if `params_cross_reg` is empty or no